libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -876,7 +876,7 @@ Vector3f AC_AttitudeControl::euler_accel_limit(const Vector3f &euler_rad, const
 }
 
 // Sets yaw target to vehicle heading and sets yaw rate to zero
-void AC_AttitudeControl::set_yaw_target_to_current_heading()
+void AC_AttitudeControl::reset_yaw_target_and_rate()
```

Comment:
It's no big deal but might as well squash this commit into, "AC_AttitudeControl: Set yaw rate to zero during arming procedure" because it's renamed in that commit as well.